src/librustc_driver/lib.rs

```diff
@@ -1412,6 +1413,7 @@ fn parse_crate_attrs<'a>(sess: &'a Session, input: &Input) -> PResult<'a, Vec<as
 /// Runs `f` in a suitable thread for running `rustc`; returns a
 /// `Result` with either the return value of `f` or -- if a panic
 /// occurs -- the panic value.
+#[cfg(not(unix))]
```

Comment:
Could a Windows solution be explored as well? It looks like the stack size can be configured in the executable headers as there is an msvc linker flag for this.

Reply:
Sounds good. Do you know a way to pass custom linker flags? It doesn't seem to be supported in build scripts.